searchcore/src/vespa/searchcore/proton/matching/matching_stats.h

```diff
@@ -82,6 +87,8 @@ struct MatchingStats
         size_t docsReRanked() const { return _docsReRanked; }
         Partition &softDoomed(bool v) { _softDoomed += v ? 1 : 0; return *this; }
         size_t softDoomed() const { return _softDoomed; }
+        Partition & doomOvertime(fastos::TimeStamp overtime) { _doomOvertime.set(overtime.sec()); return *this; }
+        fastos::TimeStamp doomOvertime() const { return fastos::TimeStamp::fromSec(_doomOvertime.max()); }
```

Comment:
would be better to make Avg non-private and let this function return Avg rather than having this function for confusion and accessing internals directly.

Reply:
I tried, but it did not look good when setting doomOvertime(time) takes a different argument than retrieving it. It think the details are best hidden like this.